DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: malariasimulation
 Title: An individual based model for malaria
-Version: 2.0.0
+Version: 2.1.0
```

Comment:
```suggestion
Version: 2.0.1
```

Reply:
Changing to 2.0.1 to signify a bug fix with no breaking changes! [Semver](https://semver.org/spec/v2.0.0.html) for reference